pkg/cli/session/session_test.go

```diff
@@ -0,0 +1 @@
+package session
```

Comment:
Probably need some tests in here @samanthajayasinghe ?

Reply:
I'll add a unit test via sub ticket since I'm planing to cover 80% this code

Reply:
:+1: